packages/studio/frontend/src/views/FlowEditor/inspector/kit/components/blockSelectors/Block/index.tsx

```diff
@@ -0,0 +1,52 @@
+import cx from 'classnames'
+import React, { useCallback, forwardRef, FC } from 'react'
+import { Grabber, OptionMenu, Text, TextIntents } from '../../../shared'
+import BlockTags from './BlockTags'
+import style from './style.scss'
+
+// export enum BlockTypes {
+//   CONTENT = 'content',
+//   CODE = 'code'
+// }
```

Comment:
Remove?

Reply:
placeholder for now, il remove once im done with this.